nix/deps/nodejs-patched/default.nix

```diff
@@ -49,34 +48,15 @@ stdenv.mkDerivation {
       ${patchMavenSources} $modBuildGradle
     done
   '';
-  # Do not add a BuildId to the generated libraries, for reproducibility
-  patchBuildIdPhase = ''
-    substituteInPlace ./node_modules/react-native/ReactAndroid/src/main/jni/Application.mk --replace \
-        '-Wl,--build-id' \
-        '-Wl,--build-id=none'
-  '';
-  # Fix bugs in Hermes usage:
-  # https://github.com/facebook/react-native/issues/25601#issuecomment-510856047
-  # - Make PR builds also count as release builds
-  # - Fix issue where hermes command is being called with same input/output file
-  patchHermesPhase = ''
-    substituteInPlace ./node_modules/react-native/react.gradle --replace \
-        'targetName.toLowerCase().contains("release")' \
-        '!targetName.toLowerCase().contains("debug")'
-  '';
-  # Patch Java files in modules which are not yet ported to AndroidX
-  patchJavaPhase = ''
-    ${nodejs}/bin/node ./node_modules/jetifier/bin/jetify
```

Comment:
Does this mean we don't need `jetifier` anymore?

Reply:
as far as I remember I had to remove this patch because this directory did not exist after upgrading.
I will look into this once more.

Reply:
yep we do not need this phase

Reply:
removed.